yarn.lock

```diff
@@ -4716,6 +4720,22 @@ puppeteer@^1.14.0, puppeteer@^1.19.0:
     rimraf "^2.6.1"
     ws "^6.1.0"
 
+puppeteer@^2.1.0:
```

Comment:
This should not come. 
Have we not added puppeteer as dev dependency in package.json of PL editor?

Reply:
puppeteer is added as a dependency and package.json was updated.

Reply:
It should be dev dependency and not direct dependency.

Reply:
I'll update it and open a new PR.

Reply:
https://github.com/publiclab/PublicLab.Editor/pull/565 here it is.

Reply:
Merged that, thank you!